main.py

```diff
@@ -44,6 +49,34 @@ def tba_webhook():
     return msg_type
 
 
+@app.route('/predictions')
+def give_predictions():
+    return str(trueskill_predictions)
+
+
+@app.route('/team_trueskill/<team>')
+def team_trueskill(team):
+    return str(trueskill.skill(team))
```

Comment:
This doesn't return plain text, and is prone to errors due to the lack of input checking. Use the API I put in instead.